WikipediaUnitTests/MWKImageFaceDetectionTests.m

```diff
@@ -96,7 +96,7 @@ - (void)testDetectingFacesInFacelessImage {
         .then(^(NSArray* faces) {
             [self.image setNormalizedFaceBoundsFromFeatures:faces inImage:facelessImage];
         });
-    } timeout:WMFDefaultExpectationTimeout WMFExpectFromHere];
+    } timeout:WMFDefaultExpectationTimeout * 2 WMFExpectFromHere];
```

Comment:
> should this be:
WMFDefaultExpectationTimeout * 2 * WMFExpectFromHere

Nope, `WMFExpectFromHere` is a macro that passes the current method & line number to the method which is used to generate an expectation description.

> Also - should we just make a longer constant?

Eventually, yes. I'd rather move towards something like:

``` objc
typedef NS_ENUM(NSUInteger, WMFTestTimeout) {
  WMFTestTimeoutShort = 0,
  WMFTestTimeoutDefault,
  WMFTestTimeoutMedium,
  WMFTestTimeoutLong
};
WMFTimeout(NSUInteger timeoutLength) {
  float constant = 0.1;
  if (getenv("TRAVIS") != NULL) {
    constant = 1;
  }
  return powf(2, timeoutLength) * 0.1;
}
```